Lib/test/test_xml_etree.py

```diff
@@ -2960,6 +2960,50 @@ def element_factory(x, y):
         del b
         gc_collect()
 
+    def test_deepcopy_clear(self):
+        # Prevent crashes when __deepcopy__() clears the children list.
+        # See https://github.com/python/cpython/issues/133009.
+        class X(ET.Element):
+            def __deepcopy__(self, memo):
+                root.clear()
+                return self
+
+        root = ET.Element('a')
+        evil = X('x')
+        root.extend([evil, ET.Element('y')])
+        if is_python_implementation():
```

Comment:
We could also make the C implementation raising RuntimeError. It is fine either way.

Reply:
I'll postpone this for a future PR.